### PR TITLE
Updates for new islandora_openseadragon.

### PR DESCRIPTION
**Pull request by adam-vessey**
Dependent on https://github.com/Islandora/islandora_openseadragon/pull/86

### CODE REVIEW

File: js/update_page.js [hunk 1]
```diff
@@ -26,25 +26,16 @@
...
-                            location.protocol + "//" + location.host + "/" +
-                            Drupal.settings.basePath + "islandora/object/" + pid
-                            + "/datastream/JP2/view?token=" + token;
-                        tile_source = new OpenSeadragon.DjatokaTileSource(
-                            settings.islandoraOpenSeadragon.settings.djatokaServerBaseURL,
-                            settings.islandoraOpenSeadragon.resourceUri,
-                            settings.islandoraOpenSeadragon
-                        );
-                        Drupal.settings.islandora_open_seadragon_viewer.open(tile_source);
+                        Drupal.settings.islandora_open_seadragon_viewer.open(data.info);
```
Comment (qadan) [inline js/update_page.js:38]: `islandora_open_seadragon_viewer` => `islandora_openseadragon_viewer`; page updater is failing with "that's not a thing"-type errors

Comment (adam-vessey) [reply js/update_page.js:38]: Update your stuff: https://github.com/Islandora/islandora_openseadragon/pull/83/files

Comment (qadan) [reply js/update_page.js:38]: Actually, that's not quite it ... pokin' this

Comment (qadan) [reply js/update_page.js:38]: Yeah, looks like possibly something up with the islandora_openseadragon_viewer theme call being behind an access call? Not investigating further

Comment (qadan) [reply js/update_page.js:38]: WELL THAT'S SILLY

**Pull request merged**